### PR TITLE
[TwigBridge] Added support for passing more files to twig:lint command

### PR DESCRIPTION
**Pull request by sustmi**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | none |
| License | MIT |
| Doc PR | none |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Command/LintCommand.php [hunk 1]
```diff
@@ -109,12 +110,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        return $this->display($input, $output, $filesInfo);
+    }
+
+    private function getFilesInfo(\Twig_Environment $twig, $filename)
+    {
         $filesInfo = array();
-        foreach ($this->findFiles($filename) as $file) {
-            $filesInfo[] = $this->validate($twig, file_get_contents($file), $file);
+
+        if (is_array($filename)) {
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: it will always be an array because of ``InputArgument::IS_ARRAY``

Comment (phansys) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: @stof, check recursive call done in line 124 (where 2nd argument is not an array).

Comment (sustmi) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: @stof: Thank you for pointing out. I had no idea. Fixed (force pushed).

File: src/Symfony/Bridge/Twig/Command/LintCommand.php [hunk 2]
```diff
@@ -94,9 +95,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $filename = $input->getArgument('filename');
+        $filenames = $input->getArgument('filename');
 
-        if (!$filename) {
+        if (count($filenames) === 0) {
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: Can you change it to `0 === count($filenames)`?

**Comment by fabpot** [comment]
:+1:

File: src/Symfony/Bridge/Twig/Command/LintCommand.php (hunk 2 above)
Comment (sustmi) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: “May the Force be with you.” ;)

**Comment by sustmi** [comment]
Fixed to `0 === count($filenames)`.

**Comment by fabpot** [comment]
Thank you @sustmi.

**Pull request merged**